client/test/extension.test.ts

```diff
@@ -1,22 +1,22 @@
-//
-// Note: This example test is leveraging the Mocha test framework.
-// Please refer to their documentation on https://mochajs.org/ for help.
-//
+// //
+// // Note: This example test is leveraging the Mocha test framework.
+// // Please refer to their documentation on https://mochajs.org/ for help.
```

Comment:
Nit: Is it really necessary to delete that here? Removing boilerplate testings not relevant to fixing packaging